test.sh

```diff
@@ -236,6 +238,17 @@ clean_up()
     fi
 }
 
+convert_notebooks()
+{
+    echo "testing notebooks"
+    for notebook in `grep ipynb docs/tutorials.rst | sed -e 's/^[ \t]*//'`
+    do
+    jupyter nbconvert --to notebook --execute "docs/$notebook"
```

Comment:
I tried to chain this command with `|| echo "Error!"` so that we just print `echo` rather than being interrupred. It did not work.

Reply:
That seems strange. So, you did:

```
jupyter nbconvert --to notebook --execute "docs/$notebook" || echo "Error!"
```

What happened? The echo part only prints if the `jupyter` command fails. If it succeeds, then nothing is echoed

Reply:
Okay... I have no idea what happend before :smile:  I probably missed the message despite me running it several times. I tried it again and I found the message printed by `echo` when the first command fails.